src/agent/application.hpp

```diff
@@ -64,6 +64,14 @@
 
 namespace otbr {
 
+#if OTBR_ENABLE_VENDOR_SERVER
+namespace vendor {
+
+class VendorServer;
```

Comment:
What's this forward declaration used for?

Reply:
I used this to fix compile issue.